command/validate.go

```diff
@@ -0,0 +1,75 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/consul/command/agent"
+	"github.com/hashicorp/consul/command/base"
+)
+
+// ValidateCommand is a Command implementation that is used to
+// verify config files
+type ValidateCommand struct {
+	base.Command
+}
+
+func (c *ValidateCommand) Help() string {
+	helpText := `
+Usage: consul validate [options] FILE_OR_DIRECTORY...
+
+  Performs a basic sanity test on Consul configuration files. For each file
+  or directory given, the validate command will attempt to parse the
+  contents just as the "consul agent" command would, and catch any errors.
+  This is useful to do a test of the configuration only, without actually
+  starting the agent.
+
+  Returns 0 if the configuration is valid, or 1 if there are problems.
+
+` + c.Command.Help()
+
+	return strings.TrimSpace(helpText)
+}
+
+func (c *ValidateCommand) Run(args []string) int {
+	var configFiles []string
+	var quiet bool
+
+	f := c.Command.NewFlagSet(c)
+	f.Var((*agent.AppendSliceValue)(&configFiles), "config-file",
```

Comment:
I left those in as hidden flags to stay compatible with configtest; everything (args, `-config-file`, `config-dir`) gets added to `configFiles` which works with files or directories. Do you think we should just remove the flags from validate altogether?